src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,22 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { React } from 'react';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => {
+  const {completed, title, user} = todo;
+
+  const cls = [
+    'TodoInfo',
+    completed ? 'TodoInfo--completed' : '',
+  ];
+
+  return (
+    <article className={cls.join(' ')}>
+      <h2 className="TodoInfo__title">{title}</h2>
```

Comment:
as variant you can do next:
```suggestion
  const completedClass = completed ? 'TodoInfo--completed' : '';

  return (
    <article className={`TodoInfo ${completedClass}`}>
      <h2 className="TodoInfo__title">{title}</h2>
```